include/net/lwm2m.h

```diff
@@ -871,12 +880,13 @@ typedef void (*lwm2m_ctx_event_cb_t)(struct lwm2m_ctx *ctx,
  *
  * @param[in] client_ctx LwM2M context
  * @param[in] ep_name Registered endpoint name
+ * @param[in] flags Flags used to configure current LwM2M session.
  * @param[in] event_cb Client event callback function
  *
  * @return 0 for success or negative in case of error.
  */
 void lwm2m_rd_client_start(struct lwm2m_ctx *client_ctx, const char *ep_name,
```

Comment:
Just noticed this, the function is void but comment says it returns 0 for success and <0 for error. Which one is correct?

Reply:
It's void, I'll remove the invalid docs.